validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -243,7 +300,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(f'Loaded {len(df)} objects from \"{catalog_name}\" with a subsampling of {sampling_factor}x')"
+    "# Choose means of obtaining data: GCR\n",
+    "df = get_GCR(catalog_name, columns, sampling_factor, snr_cut, tract)\n",
+    "#     or Postgres\n",
+    "#df = get_pg(schema_name, columns, sampling_factor, snr_cut,tract)"
```

Comment:
When I ran each of these lines I got a different number of objects returned:
```
Querying catalog dc2_object_run2.1i_dr1_tract3830
Loaded 3914 objects from "dc2_object_run2.1i_dr1_tract3830" with a subsampling of 100x
CPU times: user 16 ms, sys: 18.5 ms, total: 34.5 ms
Wall time: 10.2 s
Loaded 3927 objects from "run21i_dr1b_v1_pg2" with a subsampling of 100x
```
And they weren't the same objects.  The `objectId`s had zero intersection.

Reply:
Did something get out of date?

Reply:
The pg routine had the wrong variable name in its modulo expression.  It was finding all the mod 2 objectids whereas the GCR code found mod 22.  They now find the same ones.